lib/digitalocean/certificate.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+  var slice = [].slice,
+    util = require('./util');
+
+  /**
+   * Certificate resource
+   * @class Certificate
+   */
+  var Certificate = (function() {
+    function Certificate(client) {
+      this.client = client;
+    }
+
+    /**
+     * List all SSL certificates.
+     *
+     * @param {(number|object)} [page or queryObject] - page number to retrieve or key value pairs of query parameters
+     * @param {number} [perPage] - number of result per page to retrieve
+     * @param {requestCallback} [callback] - callback that handles the response
+     * @memberof Certificate
+     */
+    Certificate.prototype.list = function() {
+      var args = util.extractListArguments(arguments, 0);
+      return this.client.get.apply(this.client, ['/certificates',
+        {}].concat(slice.call(args.params), [200, 'certificates',
+        args.callback]));
+    };
+
+    /**
+     * Create a certificate object.
+     *
+     * @param {object} attributes - The attributes with which to create the domain. See the {@link https://developers.digitalocean.com/documentation/v2/#domains|official docs} for valid attributes.
```

Comment:
Looks like this link should be `https://developers.digitalocean.com/documentation/v2/#certificates`?